### PR TITLE
ops-bedrock:modify the script to support fast start devnet

### PR DESCRIPTION
**Pull request by welkin22**
Users can quickly deploy the bsc chain and opbnb chain by using the command `make devnet-up`.
On the bsc chain, there is an init holder with a large amount of BNB, the address is: 0x04d63aBCd2b9b1baa327f2Dda0f873F197ccd186

### CODE REVIEW

**Comment by welkin22** [comment]
user guide doc： https://github.com/bnb-chain/opbnb-docs/pull/21  
https://docs.bnbchain.org/opbnb-docs/docs/tutorials/running-a-local-development-environment

File: bedrock-devnet/devnet/__init__.py [hunk 1]
```diff
@@ -90,6 +88,27 @@ def main():
     if os.path.exists(genesis_l2_path):
         log.info('L2 genesis and rollup configs already generated.')
     else:
+        log.info('Generating network config.')
+        shutil.copy(devnet_cfg_orig, devnet_cfg_backup)
+        deploy_config = read_json(devnet_cfg_orig)
+        l1BlockTag = l1BlockTagGet()["result"]
+        print(l1BlockTag)
```
Comment (owen-reorg) [inline bedrock-devnet/devnet/__init__.py:95]: shall we use `log.info` here?

Comment (welkin22) [reply bedrock-devnet/devnet/__init__.py:95]: yes,I will fix it

File: Makefile [hunk 2]
```diff
@@ -66,7 +66,8 @@ nuke: clean devnet-clean
 .PHONY: nuke
 
 devnet-up:
-	@bash ./ops-bedrock/devnet-up.sh
+	pip3 install python-dotenv
```
Comment (owen-reorg) [inline Makefile:69]:
```suggestion
	pip3 install python-dotenv requests
```

**Comment by welkin22** [comment]
Please see https://github.com/bnb-chain/opbnb/pull/73, this is outdated

**Pull request closed without merging**